### PR TITLE
add pytest

### PR DESCRIPTION
**Pull request by dinithiW**

### CODE REVIEW

**Review by mr-c — CHANGES_REQUESTED** [review]
Can you look over https://docs.pytest.org/en/6.2.x/goodpractices.html#choosing-a-test-layout-import-rules so that you don't need to move the python files?

**Comment by mr-c** [comment]
Lets name the package `wdl2cwl`

**Comment by mr-c** [comment]
When you're done with the PR, I'll setup https://en.wikipedia.org/wiki/Continuous_integration with GitHub Actions to automatically run pytest on every PR after this one

File: tests/WdlV1_1.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+index = 0
+for i in command_arguments:
+    if "~" in i:
+        parameter_reference = i[i.find("~{")+2:i.find("}")] #finding the name of the parameter
+        sub_str = i.strip().split("~")
+        if "INPUT" not in sub_str[0]:
+            command_arguments[index] = sub_str[0]+"$(inputs."+parameter_reference+")"
+        else:
+            command_arguments[index] = sub_str[0]+"$(inputs."+parameter_reference+".path"")"
+    index+=1
```
Comment (mr-c) [inline tests/WdlV1_1.py:51]: Move this into the `main` function. Add an argument for the file name of the input WDL file

**Comment by dinithiW** [comment]
> Lets name the package `wdl2cwl` and then follow https://docs.pytest.org/en/6.2.x/goodpractices.html#tests-as-part-of-application-code

@mr-c Do we have to change the project name to wdl2cwl as well? Anyway, having the name as wdl-cwl-translator gives a syntax error when used for imports

**Comment by mr-c** [comment]
> > Lets name the package `wdl2cwl` and then follow https://docs.pytest.org/en/6.2.x/goodpractices.html#tests-as-part-of-application-code
> 
> @mr-c Do we have to change the project name to wdl2cwl as well? Anyway, having the name as wdl-cwl-translator gives a syntax error when used for imports

That makes sense, the `-` will be interpreted as a minus sign. So either use `wdl_cwl_translator` or `wdl2cwl`

**Review by mr-c — APPROVED** [review]

**Pull request merged**